### PR TITLE
Python3 - just testing the water

### PR DESCRIPTION
**Pull request by sbc100**

### CODE REVIEW

**Comment by binji** [comment]
I've been trying to keep things 2/3 compatible. AFAIK it's mostly there, since I do a bunch of testing on my arch laptop (which defaults to python3)

**Comment by sbc100** [comment]
I know .. but I've been wanting to experiment with type annotations.      And I was kinda thinking, why do we need python2 compat here?

**Comment by binji** [comment]
Ah, I see.

File: CMakeLists.txt [hunk 1]
```diff
@@ -373,7 +373,7 @@ if (NOT EMSCRIPTEN)
   endif ()
 
   # test running
-  find_package(PythonInterp 2.7 REQUIRED)
+  find_package(PythonInterp 3.4 REQUIRED)
```
Comment (rathann) [inline CMakeLists.txt]: A better change would be:
``` diff
-    find_package(PythonInterp 2.7 REQUIRED)
+    find_package(Python COMPONENTS Interpreter 3.4 REQUIRED)
```
or even

``` diff
-    find_package(PythonInterp 2.7 REQUIRED)
+    find_package(Python3 COMPONENTS Interpreter 3.4 REQUIRED)
```

See https://cmake.org/cmake/help/latest/module/FindPython.html and https://cmake.org/cmake/help/latest/module/FindPython3.html .

**Comment by binji** [comment]
This is pretty old, and we have python3 testing on CI now. Let's close this.

**Comment by sbc100** [comment]
But this change was to drop support for python2.. which allows us to cleanup some stuff.

File: CMakeLists.txt (hunk 1 above)
Comment (sbc100) [reply CMakeLists.txt]: I think I tried that and ran into some issues.  Can do leave that for a followup?

**Comment by sbc100** [comment]
Reopened as https://github.com/WebAssembly/wabt/pull/1321

**Pull request closed without merging**